src/metatensor/models/experimental/alchemical_model/tests/test_regression.py

```diff
@@ -70,7 +71,7 @@ def test_regression_init():
     expected_output = torch.tensor([[-1.9819], [0.1507], [1.6116], [3.4118], [0.8383]])
 
     torch.testing.assert_close(
-        output["U0"].block().values, expected_output, rtol=1e-05, atol=1e-4
+        output["mtm::U0"].block().values, expected_output, rtol=1e-05, atol=1e-4
```

Comment:
You could also change it to `energy` right?

Reply:
Right. We're using energy in other tests, so I would keep this as it is to check that custom names also work

Reply:
Yes works for me, Or you even do a `pytest.mark.parametrize("key", ["energy", "mtm::U0"])` to check that the `::` syntax works as well here.